tls/s2n_record_write.c

```diff
@@ -521,6 +542,12 @@ int s2n_record_writev(struct s2n_connection *conn, uint8_t content_type, const s
     return data_bytes_to_take;
 }
 
+/*
+ *= https://tools.ietf.org/rfc/rfc8446#5.1
+ *# Zero-length
+ *# fragments of Application Data MAY be sent, as they are potentially
+ *# useful as a traffic analysis countermeasure.
+ */
```

Comment:
I'm wondering if there's a better place for this comment? It seems like s2n_record_write will work when sending zero-length application data records, but this doesn't seem possible to actually do this with our API. Would this comment be better as an exception maybe?